src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -141,7 +152,7 @@ const SidebarView = View.extend({
     this.showChildView('save', new DisabledSaveView());
   },
   showForm() {
-    if (this.model.isNew()) {
+    if (this.model.isNew() || !this.model.get('last_active_at')) {
```

Comment:
are the fields read-only if they can't be edited?  They shouldn't look like they can be edited.